<launch>
  <arg name="display_windows" default="true" />

  <node name="robot_node" pkg="tiles_loc" type="base" output="screen" >
    <param name="display_window" type="bool" value="$(arg display_windows)"/>
    <param name="pos_x_init" type="double" value="0"/>
    <param name="pos_y_init" type="double" value="0"/>
    <param name="pos_th_init" type="double" value="0"/>
  </node>

  <node name="localization_node" pkg="tiles_loc" type="loc" output="screen">
    <param name="pos_x_init" type="double" value="0"/>
    <param name="pos_y_init" type="double" value="0"/>
    <param name="pos_th_init" type="double" value="0"/>
  </node>

  <node name="viewer_node" pkg="tiles_loc" type="view" output="screen" />
</launch>
